src/content/docs/en/guides/upgrade-to/v1.mdx

```diff
@@ -592,8 +592,6 @@ In Astro v0.21+, import aliases can be added from `tsconfig.json` or `jsconfig.j
 }
 ```
 
-_These aliases are integrated automatically into [VSCode](https://code.visualstudio.com/docs/languages/jsconfig) and other editors._
```

Comment:
@sarah11918 Should we keep this line also?

Reply:
Yeah, I was thinking less necessary there!  🙌